pkg/commands/test/test.go

```diff
@@ -219,35 +193,35 @@ func makeQuery(rule string) string {
 	return fmt.Sprintf("data.%s.%s", viper.GetString("namespace"), rule)
 }
 
-func processData(ctx context.Context, input interface{}, compiler *ast.Compiler) (checkResult, error) {
+func processData(context context.Context, input interface{}, compiler *ast.Compiler) (CheckResult, error) {
```

Comment:
@Salamandastron1 lets change the name of this argument from `context` to `ctx` so there isnt a name conflict with the package